etna/transforms/missing_values/imputation.py

```diff
@@ -82,204 +93,155 @@ def __init__(
         ValueError:
             if incorrect strategy given
         """
+        super().__init__(required_features=[in_column])
         self.in_column = in_column
-        self.strategy = ImputerMode(strategy)
+        self.strategy = strategy
```

Comment:
Why we need to store this? Seems like this field is not used

Reply:
It is more safe to store inital values for `to_dict` method. Because if we store `ImputerMode` value in `self.strategy` after `to_dict` it will turn into big string representation from which it can't be recreated.

We should probably create a more general solution for enums.

Reply:
Got it